packages/fiori/src/Timeline.ts

```diff
@@ -371,6 +388,60 @@ class Timeline extends UI5Element {
 		}
 	}
 
+	_handleDown() {
+		if (!this.growsWithButton) {
+			return;
+		}
+
+		this._shouldFocusGrowingButton();
```

Comment:
This sounds like a function that returns a boolean.

Also, no need for early return if there is only 1 line of code, so change to:

```ts
if (this.growsWithButton) {
		this.focusGrowingButton();
}		
```

and then merge `_shouldFocusGrowingButton` and `focusGrowingButton`